### PR TITLE
fix: Issue #452 - Show Error Message, If Transfer request failed

### LINKED ISSUE #452: Show Error Message, If Transfer request failed
**Summary:**

Whenever a user is making any type of Transfer and If the request failed due to some reasons like No Internet or Socket Exception etc. It's showing empty Snack Bar without a message. It should show message `Unable to make Transfer, Please Transfer later`

**Steps to reproduce:**

Go to Transfer in Navigation and enter details and disconnect the internet.

**Expected behavior:**

It should show a proper message, So the user should know what happened, why a user can't make transfer right now.

**Observed behavior:**

It's showing empty Snack Bar

**Device and Android version:**

Moto G

### PR DESCRIPTION
**Pull request by luckyman20**
Fixes #452 
@mdew192837 Please review it!!

### CODE REVIEW

**Review by Jdew192837 — COMMENTED** [review]
Let's make the message "Unable to complete transfer, please try again later".

I think it's more clear and less redundant

**Comment by luckyman20** [comment]
@Jdew192837 I have just copied this message from the issue itself.
Please check issue also

**Comment by mdew192837** [comment]
@therajanmaurya Can you dismiss that review? I made it on the wrong account

**Comment by therajanmaurya** [comment]
First It's ok @mdew192837, Yeah @mdew192837 the message you suggested that is good too.
@luckyman20 make the message value change and let me know.

**Comment by luckyman20** [comment]
@therajanmaurya @mdew192837 Made the changes

File: app/src/main/res/values-hi/strings.xml [hunk 1]
```diff
@@ -110,6 +110,7 @@
     <string name="empty_repayment_schedule">आपके साथ संबद्ध कोई भी पुनर्भुगतान शेड्यूल नहीं है</string>
     <string name="no_more_transactions_available">कोई और लेनदेन उपलब्ध नहीं है</string>
     <string name="no_transaction">कोई लेनदेन नहीं</string>
+    <string name="transfer_error">स्थानांतरण पूर्ण करने में असमर्थ, कृपया बाद में फिर से प्रयास करेंं</string>
```
Comment (mdew192837) [inline app/src/main/res/values-hi/strings.xml:113]: What language is this, just curious? Just remember that unless this is your native language, by GCI rules, you are not allow to add translations. :)

Comment (luckyman20) [reply app/src/main/res/values-hi/strings.xml:113]: @mdew192837 This language is Hindi, and it is needed to be added as Mifos Mobile App gives two language options Hindi and English as of now

**Comment by mdew192837** [comment]
Does this look good? @therajanmaurya

File: app/src/main/res/values-hi/strings.xml (hunk 1 above)
Comment (therajanmaurya) [reply app/src/main/res/values-hi/strings.xml:113]: @mdew192837 we are supporting Hindi too.

**Pull request merged**